.eslintrc.js

```diff
@@ -10,7 +10,4 @@ module.exports = {
     'monaco-editor.d.ts',
     'next-env.d.ts',
   ],
-  parserOptions: {
-    project: `${__dirname}/tsconfig.json`,
-  },
```

Comment:
[`parserOptions.project: true`](https://typescript-eslint.io/blog/parser-options-project-true/) would have made this automatic. But [`parserOptions.projectService`](https://typescript-eslint.io/blog/announcing-typescript-eslint-v8#project-service) is even shinier. ✨